docs/_toc.yml

```diff
@@ -35,9 +40,10 @@ parts:
     - file: content/figures
     - file: content/metadata
 
-  - file: execute/index
+  - file: content/executable/index
```

Comment:
I'd like to re-organize these files so they follow a proper folder sub-structure, but we'll need to create redirects for this, and I don't wanna do it in this PR because it'll make the review a big pain